benchmarks/pvc/gemm_configuration.hpp

```diff
@@ -111,10 +114,29 @@ struct GemmConfiguration<
   using GemmKernel = kernel::GemmUniversal<
     Shape<int, int, int, int>,
     CollectiveMainloop,
-    CollectiveEpilogue
+    CollectiveEpilogue,
+    std::conditional_t<TileScheduler == Scheduler::Parallel, void, cutlass::gemm::StreamKScheduler>
   >;
 
   using Gemm = GemmUniversalAdapter<GemmKernel>;
+
+  constexpr static typename GemmKernel::Arguments defaultArguments() {
+    using StreamKMode =
+      cutlass::gemm::kernel::detail::PersistentTileSchedulerXeStreamKParams::DecompositionMode;
+    if constexpr (TileScheduler == Scheduler::Parallel) {
+      return {};
+    }
+    else if constexpr (TileScheduler == Scheduler::StreamK) {
+      typename GemmKernel::Arguments arguments{};
+      arguments.scheduler = {1, StreamKMode::StreamK};
+      return arguments;
+    }
+    else if constexpr (TileScheduler == Scheduler::SplitK) {
+      typename GemmKernel::Arguments arguments{};
+      arguments.scheduler = {1, StreamKMode::SplitK};
+      return arguments;
+    }
+  }
```

Comment:
Total nit: Could be worth asserting we never drop out the end of this `if-else-else`? Obviously at the moment it's impossible given the enum def, but for future proofing?

Reply:
done